src/Shopsys/ShopBundle/Model/Product/Pricing/ProductPriceCalculationForUser.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Shopsys\ShopBundle\Model\Product\Pricing;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Customer\CurrentCustomer;
+use Shopsys\FrameworkBundle\Model\Customer\User;
+use Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroupSettingFacade;
+use Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceCalculation;
+use Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceCalculationForUser as BaseProductPriceCalculationForUser;
+use Shopsys\FrameworkBundle\Model\Product\Product;
```

Comment:
Please change `FrameworkBundle` to `ShopBundle` in usages and annotations of classes that are extended in `ShopBundle`